homepage/homepage/content/docs/reference/testing.mdx

```diff
@@ -0,0 +1,214 @@
+import { Alert, CodeGroup, ContentByFramework } from '@/components/forMdx';
+
+export const metadata = {
+  description: "Guidance on testing Jazz apps."
+};
+
+# Testing Jazz Apps
+
+As you develop your Jazz app, you might find yourself needing to test functionality relating to sync, identities, and offline behaviour. The `jazz-tools/testing` utilities provide helpers to enable you to do so.
+
+## Core test helpers
+Jazz provides some key helpers that you can use to simplify writing complex tests for your app's functionality.
+
+### `setupJazzTestSync`
+
+This should normally be the first thing you call in your test setup, for example in a `beforeEach` or `beforeAll` block. This function creates and returns a plain `Account`, which will also act as an in-memory sync node for the test session. You'll often pair this with `createJazzTestAccount()` for more complex testing. Test data is not persisted, and no clean-up is needed between test runs. 
+
+<CodeGroup>
+```ts twoslash
+import { co } from 'jazz-tools';
+import { beforeEach, describe, expect, test } from "vitest";
+import { createJazzTestAccount, setupJazzTestSync } from 'jazz-tools/testing';
+const MyAccountSchema = co.account({
+  profile: co.profile(),
+  root: co.map({})
+});
+// ---cut---
+describe("My app's tests", () => {
+  beforeEach(async () => {
+    await setupJazzTestSync();
+  });
+
+  test("I can create a test account", async () => {
+    // See below for details on createJazzTestAccount()
+    const account1 = await createJazzTestAccount({
+      AccountSchema: MyAccountSchema,
+      isCurrentActiveAccount: true
+    });
+    expect(account1).not.toBeUndefined();
+    // ...
+  });
+});
+```
+</CodeGroup>
+
+<Alert variant="info" title="Important" className="mt-4">
+  The account returned by `setupJazzTestSync` is not created based on any custom schemas you may have defined, and no migrations will be run on this account.
```

Comment:
The account value returned by `setupJazzTestSync` is only meant to be used to check the status of CoValues on the sync node.

It's a pretty advanced use-case, we could remove it if you think it might create confusion.